BedrockServer.cpp

```diff
@@ -1340,7 +1340,7 @@ void BedrockServer::_network(BedrockServer& server) {
                 // We'll get interrupted if either work becomes available or we're supposed to exit, so just start at
                 // the top of the loop if that happens.
                 auto beforeWait = chrono::steady_clock::now();
-                server._networkCV.wait(lock);
+                server._networkCV.wait_for(lock, 1s);
```

Comment:
Doesn't really matter except if nothing's happening (the command port is closed), this will sit and not log the 10s stats, and when it does log the timing will be weird (can be for a lot longer than 10s)

Reply:
oh that makes sense